runtime/sema/occurrences.go

```diff
@@ -96,6 +97,13 @@ func (o *Occurrences) Put(startPos, endPos ast.Position, origin *Origin) {
 		occurrence.EndPos,
 	)
 	o.tree.Put(interval, occurrence)
+	origin.Occurrences = append(
+		origin.Occurrences,
+		ast.Range{
+			StartPos: startPos,
+			EndPos:   endPos,
+		},
+	)
```

Comment:
The origin may be nil, e.g. for built-in members:

```suggestion
	if origin != nil {
		origin.Occurrences = append(
			origin.Occurrences,
			ast.Range{
				StartPos: startPos,
				EndPos:   endPos,
			},
		)
	}
```